sdk-lib/src/main/java/cash/z/ecc/android/sdk/model/TransactionOutput.kt

```diff
@@ -0,0 +1,11 @@
+package cash.z.ecc.android.sdk.model
+
+data class TransactionOutput(
```

Comment:
Nit, `TransactionOutput` is badly named here; a transaction output is a transparent UTXO or a shielded note, this is _only_ the output pool data. Why not use `TransactionPool` directly?

Reply:
Unless the plan is to add more information to this type, I agree with @nuttycom .